politeiawww/cmd/shared/config.go

```diff
@@ -12,19 +12,20 @@ import (
 	"net/http"
 	"net/url"
 	"os"
-	"os/user"
 	"path/filepath"
 	"runtime"
 	"strings"
 
 	"github.com/decred/dcrd/dcrutil/v3"
 	"github.com/decred/politeia/politeiad/api/v1/identity"
+	"github.com/decred/politeia/politeiawww/config"
+	"github.com/decred/politeia/util"
 	"github.com/decred/politeia/util/version"
 	flags "github.com/jessevdk/go-flags"
 )
 
 const (
-	defaultHost              = "https://proposals.decred.org/api"
+	defaultHost              = "https://127.0.0.1:4443"
```

Comment:
Is this right?

Reply:
Yes, these are dev tools and not meant for production, so we default to localhost.  Defaulting to proposals.decred.org just adds in one more step when someone is trying to use it because then they have to set it to localhost.